subiquity/ui/mount.py

```diff
@@ -79,6 +79,16 @@ def enable_common_mountpoints(self):
             if isinstance(opt.value, str):
                 opt.enabled = opt.value in common_mountpoints
 
+    def disable_all_mountpoints_but_home(self):
+        """Currently, filesystems of non-local disks can only be mounted at
+        /home."""
```

Comment:
The comment isn't quite right as users can type whatever they like into the "other" box, including things like / and /boot. Is it worth displaying a warning like we do when the user mounts a partition at say / without reformatting it?

Reply:
Good point, updated the comment and added a warning message:

![2024-01-26T14:52:04,155473089+01:00](https://github.com/canonical/subiquity/assets/4038023/9b1909e3-5ae3-4d64-8462-53425e75b1e2)